packages/components/combobox/src/combobox.ts

```diff
@@ -215,8 +260,53 @@ export class Combobox<T = unknown> extends FormControlMixin(ScopedElementsMixin(
   override willUpdate(changes: PropertyValues<this>): void {
     super.willUpdate(changes);
 
-    if (changes.has('options') || changes.has('value')) {
-      this.#updateSelected();
+    if (changes.has('multiple')) {
+      if (this.multiple) {
```

Comment:
Maybe
`this.listbox?.toggleAttribute('aria-multiselectable', this.multiple);
`
would be simpler?

Reply:
No, that would make it a `boolean` and it needs to be a `string`